Source/Urho3D/Graphics/GraphicsDefs.h

```diff
@@ -79,6 +79,7 @@ enum BlendMode
     BLEND_INVDESTALPHA,
     BLEND_SUBTRACT,
     BLEND_SUBTRACTALPHA,
+    BLEND_DECAL,
```

Comment:
This name sucks but I cannot think of better one. Let's call it `BLEND_DEFERRED_DECAL` at least